src/index.js

```diff
@@ -0,0 +1,29 @@
+import React              from "react";
+import ReactDOM           from "react-dom";
+import {createStore}      from "redux";
+import {Provider}         from "react-redux";
+import App                from "./components/App";
+import reducer from './reducers/index';
+import * as serviceWorker from "./serviceWorker";
+
+export const initialState = {
+  filter: 'All',
+  books: [
+      {id: Math.random(), title: 'The Narrative of Arthur Gordon Pym of Nantucket', category: 'Horror'},
```

Comment:
`Math.random()` returns a value between 0 a 1 which doesn't seem to be a good candidate for ID.

Reply:
Required by the project guideline. If you ask me ID can't be random ever.

Reply:
You're right. Let's leave it as is. We'll correct requirements for future students :)